content/v3/pulls/comments.md

```diff
@@ -62,7 +62,7 @@ Name | Type | Description
 Name | Type | Description
 -----|------|--------------
 `body`|`string` | **Required**. The text of the comment
-`commit_id`|`string` | **Required**. The SHA of the commit to comment on.
+`commit_id`|`string` | **Required**. The SHA of the latest commit of the pull request to comment on.
```

Comment:
> I dont know how to make a note.

There's nothing special to a note, just a body of text prefaced by the word "Note" :smile:. [Here](https://developer.github.com/v3/repos/#list-all-public-repositories) is one such example of what it could look like.